massa-pool/src/tests/scenario.rs

```diff
@@ -183,6 +184,172 @@ async fn test_pool() {
     )
     .await;
 }
+
+#[tokio::test]
+#[serial]
+async fn test_pool_with_execute_sc() {
+    let (cfg, thread_count, operation_validity_periods, max_pool_size_per_thread): &(
+        PoolSettings,
+        u8,
+        u64,
+        u64,
+    ) = &POOL_SETTINGS;
```

Comment:
Is there a need to assign these? I think it would be clearer to just use  `POOL_SETTINGS` directly where it is needed.

Reply:
Indeed, I'll fix it elsewhere too

Reply:
Ok so i'm not exactly sure why but POOL_SETTINGS returns the settings and other non pool related config stuff